pkgs/data/fonts/iosevka/package-lock.json

```diff
@@ -1,62 +1,286 @@
 {
```

Comment:
I noticed this file got much larger. This is the only `package-lock.json` file in Nixpkgs; what are other Node packages doing to avoid this?

Reply:
I will try removing it

Reply:
It seems that they are using node2nix: https://github.com/svanderburg/node2nix
Exemple here: https://github.com/NixOS/nixpkgs/blob/40d59c6e8e1f179822f6f78a221a097aca3169d1/pkgs/tools/networking/airfield/deps.sh
Or here: https://github.com/NixOS/nixpkgs/blob/40d59c6e8e1f179822f6f78a221a097aca3169d1/pkgs/tools/security/bitwarden-cli/generate.sh

Should I make the migration too ?

Reply:
I've pushed the new version with the usage node2nix.